embdgen-core/src/embdgen/core/utils/SizeType.py

```diff
@@ -1,18 +1,16 @@
 # SPDX-License-Identifier: GPL-3.0-only
 
-from typing import Optional
+from __future__ import annotations
+from typing import Any, Optional
 import re
 
-BYTES_PER_SECTOR = 512
+BYTES_PER_SECTOR = 0x200
+MULTIPLIER = {"k": 0x400, "M": 0x100000, "G": 0x40000000}
```

Comment:
I don't think this increases readability. A sector is always defined in decimal, you will always find 512 bytes
A kB is  defined as 1024 bytes, not 0x400 bytes, and so is the MB, ...

Reply:
Hexadecimal numbers are much easier to read, because they provide a human-friendly representation of binary-coded values. And no, I don't think that 1073741824 is better than 0x40000000, sorry. For users decimal might be better, but hex are always easier: 0x200, 0x400, 0x800, 0x1000...

Reply:
No hex is not always easier, if the default for a value used is decimal.